.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+
+name: RELEASE
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  perform-release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Get the version
+        id: get_version
+        run: echo ::set-output name=VERSION::$(echo ${GITHUB_REF##*/})
+
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Publish artifact
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          RELEASE_VERSION: ${{ steps.get_version.outputs.VERSION }}
```

Comment:
Nice!

Reply:
Haha. It took a while to figure out, but it works!